### ISSUE TITLE
ReadMe tells how to install and configure but not how to use it

### ISSUE THREAD

**Issue by batisteo** [Shougo/deoplete.nvim#1151]
Hey, it's a more a documentation rather that something else, but I successfully installed the plugin, but I don't know how to accept the presented options.

**Comment by Shougo** [comment]
I don't understand.

You don't need to configure options if not needed.
You can improve the documentation to how to use it.

The PR is wellcome.

**Comment by batisteo** [comment]
- @batisteo: I don't know how to use it, there's no docs.
- @Shougo: I don't understand. Write docs.

**Comment by Shougo** [comment]
This is open source project.
I don't need to write the documentation for you.

**Comment by Shougo** [comment]
If you don't understand the usage, you can use other plugins.

**Comment by batisteo** [comment]
Allright, that's another level of RTFM.  
Are you Ok? Do you want to talk about it?

**Comment by yuki-ycino** [comment]
Please read https://github.com/Shougo/deoplete.nvim/blob/master/doc/deoplete.txt

**Comment by batisteo** [comment]
Thanks, that's useful!

**Comment by batisteo** [comment]
Just FYI, I was expected a _Usage_ section in the ReadMe, even with a link to the docs. That's my expectation, not your duty. 
 
I'm quite stuck in the “Level 1” of Vim, that I got with the [Openvim tutorial](https://www.openvim.com/) and recently I decided to reduce my usage of Electron-based editor in favor in NeoVim, and started watching some Youtube videos, one of them advising this plugin, and reading this great page [Learn vim For the Last Time: A Tutorial and Primer](https://danielmiessler.com/study/vim/).

Seems like none of these resources mention the `help` command nor how to display the docs of a command or a plugin.
